SU2_CFD/include/drivers/CDiscAdjMultizoneDriver.hpp

```diff
@@ -169,9 +169,9 @@ class CDiscAdjMultizoneDriver : public CMultizoneDriver {
   /*!
    * \brief Evaluate sensitivites for the current adjoint solution and output files.
    * \param[in] Iter - Current outer or time iteration.
-   * \param[in] StopCalc - Final iteration flag (converged or reached max number of iters).
+   * \param[in] force_writing - Force file output.
    */
-  void EvaluateSensitivities(unsigned long Iter, bool StopCalc);
+  void EvaluateSensitivities(unsigned long Iter, bool force_writing);
```

Comment:
👍 `StopCalc` is  a visible  class boolean so it shouldn't be used as a member var in the same class anyway